src/v3/src/util/settingsUtils.ts

```diff
@@ -134,7 +134,7 @@ export const getCustomHelpLinks = (widgetProps: WidgetProps): CustomLink[] => {
 
 export const getFactorPageCustomLink = (widgetProps: WidgetProps): Omit<CustomLink, 'target'> | undefined => {
   const { factorPage } = widgetProps.helpLinks || {};
-  return factorPage;
+  return typeof factorPage !== 'undefined' && 'href' in factorPage ? factorPage : undefined;
```

Comment:
This was an issue found while testing, prior to the fix, it added an empty div to the UI because we were not properly checking if the `factorPage` property was provided.

Reply:
just curious what was it returning before if `factorPage` prop was not provided?  Was there sometimes a valid `factorPage` object but with no `href` prop?

Reply:
Prior to this change, it was actually returning an empty object, then, [this](https://github.com/okta/okta-signin-widget/blob/master/src/v3/src/transformer/button/transformFactorPageCustomLink.ts#L35-L50) code which relied on that property only checked if it was set or not (because it expected it to be undefined if the factorPage property wasn't set). I only added the additional check for the href to ensure we have a valid target to redirect them to. Based on the docs, the href is required so there shouldn't be a scenario where it isn't provided but added this check to be safe (because the config options are exposed to external customers for use in embedded flows too).

Reply:
okay got it, awesome nice catch